deployments/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -51,6 +52,8 @@ spec:
               value: "nsmgr.nsm-system"
             - name: NSREGISTRY_PORT
               value: "5000"
+            - name: INSECURE
+              value: "false"
```

Comment:
I would have to make sure the operator has this value and that it can be provided by Config.